backend/workflow_manager/workflow/execution.py

```diff
@@ -305,7 +299,7 @@ def execute_input_file(
         file_name: str,
         single_step: bool,
         file_history: Optional[FileHistory] = None,
-    ) -> None:
+    ) -> bool:
```

Comment:
NIT: Add a docstring for the return type - I notice that even though a file was executed and returned from cache, we'll return false here. I assume its to indicate whether it was processed or not rather

Reply:
addressed